cfme/fixtures/pytest_selenium.py

```diff
@@ -1168,8 +1169,8 @@ def _login_func():
             recycle = True
         # If the page is blocked, then recycle...
         if (
-                is_displayed("//div[@id='blocker_div' or @id='notification']", _no_deeper=True)
-                or is_displayed(".modal-backdrop.fade.in", _no_deeper=True)):
+                is_displayed("//div[@id='blocker_div' or @id='notification']", _no_deeper=True) or
```

Comment:
@RonnyPfannschmidt This is much faster and atomic. Only one selenium call instead of two.